nupic/algorithms/CLAClassifier.py

```diff
@@ -213,6 +213,31 @@ def __setstate__(self, state):
     self._version = BitHistory.__VERSION__
 
 
+  def write(self, proto):
+    proto.id = self._id
+
+    statsProto = proto.init("stats", len(self._stats))
```

Comment:
I don't believe you need to manually init `stats` and enumerate.  There's a performance hit, so if we can avoid the enumeration, that would be better.